### PR TITLE
I added the style to the contact section and the footer

### PR DESCRIPTION
**Pull request by JesusPachdez**
hi dude! I finished the website in desktop version. 

1.- First of all I changed the backgroud linear-gradiante and the background color.

2.- I added new classes.

3.-  I changed the absolute length units to relative length units in the entire CSS style.

4.- I added style in the contact section and in the footer.

This is de new backgroud linear-gradiante and the background color in the main section:

image

This is the contact section with CSS style:

image

this is the footer with CSS style: 

image

And that's it for now, thanks in advance dude :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,38 +46,49 @@ <h1><span>ARCA:</span> INSTITUTO DE <br> PSICOTERAPIA Y ANÁLISIS DE LA CONDUCTA
...
+                <p>
+                    Horario: De lunes a viernes de 9am a 8pm y sabados de 9 am a 1 pm.
+                </p>
+            </div>
+            <div class="contact-form">
+                <div class="map">
+                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3732.983489813214!2d-103.38846888468208!3d20.67025068619371!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8428ae73c0c40f69%3A0x15888e709780a11e!2sNicol%C3%A1s%20Puga%2090%2C%20Vallarta%2C%2044690%20Guadalajara%2C%20Jal.!5e0!3m2!1ses-419!2smx!4v1644873647103!5m2!1ses-419!2smx" width="400" height="300" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
+                </div>
+                <form>
+                    <p>
```
Comment (diegonava6) [inline index.html:62]: I suggest to use a div instead of <p>

<p> is meant for put text inside but not other html elements

File: style.css [hunk 2]
```diff
@@ -8,69 +8,78 @@
...
+    padding: 1.5rem 0.5rem 1rem 0.5rem;
+    margin: 0 auto;
+    overflow: hidden;
 }
 
 .menu{
     display: flex;
-    width: 50%;
-    justify-content: space-around;
+    width: 70%;
```
Comment (diegonava6) [inline style.css:39]: check the responsiveness on different screen sizes and mobile

File: style.css [hunk 3]
```diff
@@ -81,39 +90,180 @@
...
+    font-weight: bolder;
+    width: 7.5rem;
+    border-radius: 1.9rem;
+    font-size: 1rem;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+.contact-form form button:hover,
+.contact-form form button:focus{
```
Comment (diegonava6) [inline style.css:215]: ahh perro, you even used pseudo-selectors :)

**Review by diegonava6 — APPROVED** [review]
Great job buddy  💯 

Just a comment suggesting a change, but is not a blocker to merge this PR

File: index.html (hunk 1 above)
Comment (JesusPachdez) [reply index.html:62]: ah ok bro,  I'm gonna change all of them

File: style.css (hunk 2 above)
Comment (JesusPachdez) [reply style.css:39]: ok bro I'm gonna work on this.

File: style.css (hunk 3 above)
Comment (JesusPachdez) [reply style.css:215]: 8)

Comment (JesusPachdez) [reply style.css:215]: =D

**Pull request merged**